test/algorithms/relational_operations/relate/relate_pointlike_xxx.cpp

```diff
@@ -92,6 +92,8 @@ void test_all()
 
```

Comment:
`xxx` stands for "any geometry" so I'll change the name to `relate_pointlike_geometry`.